docker.Makefile

```diff
@@ -8,15 +8,15 @@ $(warning WARNING: No docker user found using results from whoami)
 DOCKER_ORG                = $(shell whoami)
 endif
 
-CUDA_VERSION              = 11.3.1
+CUDA_VERSION              = 11.6.1
 CUDNN_VERSION             = 8
 BASE_RUNTIME              = ubuntu:18.04
 BASE_DEVEL                = nvidia/cuda:$(CUDA_VERSION)-cudnn$(CUDNN_VERSION)-devel-ubuntu18.04
 
 # The conda channel to use to install cudatoolkit
 CUDA_CHANNEL              = nvidia
 # The conda channel to use to install pytorch / torchvision
-INSTALL_CHANNEL          ?= pytorch
+INSTALL_CHANNEL          ?= pytorch-nightly
```

Comment:
oh wait we shouldn't change the defaults for this

Reply:
There might be users who rely on the default behavior so changing this could be _very_ problematic

Reply:
The issue with pytorch channel is that it does not contain pytorch-cuda yet. I think publishing pytorch-cuda package to pytorch channel should fix this issue.